src/main/java/org/mockito/invocation/Invocation.java

```diff
@@ -47,6 +50,13 @@
      */
     Object[] getRawArguments();
 
+    /**
+     * Returns arguments wrapped into ArgumentMatchers
+     *
+     * @return a list of ArgumentMatcher wrapping each of this invocation arguments
+     */
+    List<ArgumentMatcher> getArgumentsAsMatchers();
```

Comment:
This is good! We can add this method to the public API because it returns a public type.

Reply:
:)